services/catalog/tests/unit/with_dbs/test_api_rpc.py

```diff
@@ -108,12 +115,49 @@ async def test_rcp_catalog_client(
         service_version=service_version,
         update={
             "name": "foo",
-            "thumbnail": None,
             "description": "bar",
-            "classifiers": None,
         },
     )
 
     assert updated.key == got.key
     assert updated.version == got.version
     assert updated.name == "foo"
+    assert updated.description == "bar"
+    assert not updated.classifiers
+
+    got = await get_service(
+        rpc_client,
+        product_name=product_name,
+        user_id=user_id,
+        service_key=service_key,
+        service_version=service_version,
+    )
+    assert got == updated
+
+
+async def test_rcp_service_not_found_error(
```

Comment:
```suggestion
async def test_rpc_service_not_found_error(
```

Reply:
and actually you use rcp instead of rpc in this file at several locations

Reply:
thx